core/lib/contract_verifier/src/tests/real.rs

```diff
@@ -0,0 +1,141 @@
+//! Tests using real compiler toolchains. Should be prepared by calling `zkstack contract-verifier init`
+//! with at least one `solc` and `zksolc` version. If there are no compilers, the tests will be ignored
+//! unless the `RUN_CONTRACT_VERIFICATION_TEST` env var is set to `true`, in which case the tests will fail.
+
+use std::{env, sync::Arc, time::Duration};
+
+use zksync_utils::bytecode::validate_bytecode;
+
+use super::*;
+
+fn assert_no_compilers_expected() {
```

Comment:
Nit: maybe the code would be a bit easier to read if it was a macro like 
```rust
macro_rules! skip_if_not_enabled {
   () => {
       if std::env::var("RUN_CONTRACT_VERIFICATION_TEST").ok().as_deref() != Some("true") {
          println!("Test is not enabled, skipping");
          return;
       }
   }
}
```

this way the logic to skip the test will not be mixed with the actual test logic.